pillarbox-monitoring-terraform/20-pillarbox-monitoring-app/data_transfer_service.tf

```
@@ -65,7 +65,7 @@ resource "aws_ecs_task_definition" "transfer_task" {
       # Environment variables for connecting to SSE and OpenSearch services
       environment = [
         {
-          name  = "PILLARBOX_MONITORING_SSE_URI"
+          name  = "PILLARBOX_MONITORING_DISPATCH_URI"
           value = "http://${aws_service_discovery_service.dispatch_service_discovery.name}.${aws_service_discovery_private_dns_namespace.service_discovery_namespace.name}:8080/events"
```

```suggestion
          value = "https://${aws_service_discovery_service.dispatch_service_discovery.name}.${aws_service_discovery_private_dns_namespace.service_discovery_namespace.name}:8080/events"
```

Can we use HTTPS here?

---

This communication occurs in the private the subnet so it shouldn't be an issue to use http here.